### ISSUE TITLE
Symetrical distal arthritis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5869]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0040311

**Bug description**
Typo in label (Symetrical)

Also, I am wondering if this is the same as 
Symmetric(al) Polyarthritis
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4453122/

I would like to make the following change

Label: Symmetric polyarthritis
Def: Polyarthritis refers to a joint disease that involves at least five joints.  One or more signs of inflammation, including pain, movement restriction, swelling, warmth, and redness, are seen in the joints involved. Polyarthritis is defined as symmetric if at least half of the joints involved are affected in a symmetric (i.e., both right and the left) fashion. 
Comment: In the event that pain is the only symptom, it is difficult to differentiate polyarthritis from the causes of polyarticular joint pain, such as fibromyalgia or osteoarthritis.
PMID:31657698

**Comment by pnrobinson** [comment]
done
